src/modules/infraestructure/repositories/custom/local.array.repository.ts

```diff
@@ -29,6 +29,16 @@ export class LocalArrayRepository implements Repository {
         }));
         return true;
     }
+
+    async update(item: Domain): Promise<Domain> {
+        let deleted: boolean = await this.delete(item);
```

Comment:
I guess this is a copy/paste :smile:

Reply:
No. To update item I delete old existing item and add new one.